Global Features blazor docs (#12773)
abpframework/abp: docs/en/UI/Blazor/Global-Features.md

# Blazor UI: Global Features

> `GlobalFeatureManager` allows you to check the features are enabled or disabled in the blazor project.

Reviewer: ```suggestion
`GlobalFeatureManager` allows you to check the global features in your Blazor applications.
```

> > This document only explains the JavaScript API. See the [Global Features](../../Global-Features.md) document to understand the ABP Global Features system.

Reviewer: You can completely remove this paragraph (this is not JavaScript already)

## Usage

```html
@using Volo.Abp.GlobalFeatures

@* ... *@

@* Global Feature can be checked with feature name *@
@if(GlobalFeatureManager.Instance.IsEnabled("Ecommerce.Subscription"))
{
    <span>Ecommerce.Subscription is enabled.</span>
}

@* OR it can be checked with type *@

@if(GlobalFeatureManager.Instance.IsEnabled<EcommerceSubscriptionGlobalFeature>())
{
    <span>Ecommerce.Subscription is enabled.</span>
}
```

> - You can follow [Check for a Global Feature](../../Global-Features#check-for-a-global-feature) section of Global Features document to check global featues in csharp.

Reviewer: ```suggestion
- You can follow _Check for a Global Feature_ section of the [Global Features document](../../Global-Features.md) to check global features in your C# code.
```
